### DISCUSSION TITLE
TP7 - Problema con el #include

### DISCUSSION THREAD

**Discussion by Nikzza** [INGCOM-UNRN-P1/cursada-2023#182] [General de la cátedra]
Para llamar a otras librerias, no funcionara llamandolas con

![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141664052/b21894a9-ffa1-4ba4-81e8-78bd6a41bbbc)

Es con comillas dobles, en el ejemplo del ejercicio esta mal colocado

```C
#include "practica3.c"
```

**Comment by Nikzza** [comment]
es con comillas dobles, en el ejemplo del ejercicio esta mal colocado

**Comment by martinvilu** [comment]
Excelente aporte, modifiqué un toque el planteo y lo deje fijo para ver.
